app/concerns/permission_checking.rb

```diff
@@ -42,7 +42,7 @@ def is_non_nasa_draft_approver?(user:, token:)
 
   # shortcut helper methods
   def approver?
-    is_non_nasa_draft_approver?(user: current_user, token: token)
+    @user_has_approver_permissions || is_non_nasa_draft_approver?(user: current_user, token: token)
```

Comment:
Aren't these 2 operands identical?

Reply:
The right side goes out to CMR to check if they have the ACL, but in some cases we will have already checked and set it.  If so, we save a call to the CMR.